src/USER-DRUDE/pair_coul_tt.h

```diff
@@ -0,0 +1,76 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#ifdef PAIR_CLASS
+
+PairStyle(coul/tt,PairCoulTT)
+
+#else
+
+#ifndef LMP_PAIR_COULTT_H
+#define LMP_PAIR_COULTT_H
+
+#include "pair.h"
+#include <vector>
```

Comment:
Is there a need to include <vector> here? I don't see it.
Please check out our updated include file conventions: https://github.com/lammps/lammps/blob/master/doc/include-file-conventions.md

Reply:
It was a mistake, sorry. I have removed `#include <vector>`.